CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,17 @@
+# Code of Conduct #
+
+## DO ##
```

Comment:
What is the purpose of these changes (# on the right, * replaced with -) in most of the files when the rendered Markdown result is identical?

Reply:
Probably for raw result, however i can agree that this PR is essentially pointless.

Reply:
Those changes were made automatically by my linter,
I apologize. I'll revert them